packages/appstore/src/modules/onboarding/onboarding.scss

```
@@ -1,11 +1,13 @@
 .onboarding-wrapper {
     display: flex;
     flex-direction: column;
+    gap: 1.5rem;
     background-color: var(--general-hover);
     height: inherit;
+
     .onboarding-header {
+        margin-left: 5%;
```

```suggestion
        margin-left: 0.5rem;
```

use `rem` instead of %
check with different rem size like